.github/workflows/ci.yaml

```diff
@@ -46,6 +46,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: sudo apt install libwayland-cursor0 libxkbcommon-dev libwayland-dev
+      - run: brew install spirv-tools
```

Comment:
wait what? this should have excluded wgpu-example-runner in the clippy run, not installing spirv-tools, right? either that or including example-runner in the clippy run

Reply:
not sure what you mean?

Did get clippy build failures on this branch on that spirv-tools was missing

Reply:
Why would you use brew on a linux job?

Reply:
@bjorn3 It's the only package manager with `spirv-tools` on linux that is available in GitHub Actions.

Reply:
> Why would you use brew on a linux job?

See also https://github.com/EmbarkStudios/rust-gpu/issues/107